src/redux/Dashboard/reducer.js

```diff
@@ -1,20 +1,25 @@
 import { Map } from 'immutable';
 import actions from './actions';
+import { SortBy } from '../../constants';
 
-const initState = new Map({
+const initState = {
   value: undefined,
-});
+  sortBy: SortBy.Ascending,
+};
 
 export default function dashboardReducer(state = initState, action) {
   switch (action.type) {
-    case actions.TAB_INDEX_CHANGED: {
-      return state.set('tabIndex', action.value);
-    }
-    case actions.SORT_ORDER_CHANGED: {
-      return state.set('sortBy', action.sortBy);
-    }
-    default: {
+    case actions.TAB_INDEX_CHANGED:
```

Comment:
I noticed this is dead code. Before merging this, unless theirs any objections, I'm going to remove this

Reply:
yup remove anything not being used.